src/block/blog-posts/index.php

```diff
@@ -78,17 +78,22 @@ function stackable_blog_posts_block_default_attributes( $attributes ) {
 
 if ( ! function_exists( 'stackable_blog_posts_post_query' ) ) {
 	function stackable_blog_posts_post_query( $attributes ) {
-		return apply_filters( 'stakckable/blog-post/post_query',
-			array(
+		$passedAttributes = array(
```

Comment:
Please use snake_case in PHP variables, function names as per WP PHP standards https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/

We use camelCase in JS only